app/src/main/java/movierating/services/AverageReviewScoreByYear.java

```diff
@@ -0,0 +1,24 @@
+package movierating.services;
+
+import movierating.models.Movie;
+import movierating.repositories.MovieRepository;
+
+public class AverageReviewScoreByYear {
```

Comment:
I dont see any relation between name `AverageReviewScoreByYear` and what it does. There is no year check.

Reply:
What dude? Didn't get you.

Reply:
Its done.